ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/SandboxIndexAndWriteService.scala

```diff
@@ -387,8 +390,8 @@ abstract class LedgerBackedIndexService(
     contractStore.lookupContractKey(submitter, key)
 
   // PartyManagementService
-  override def getParticipantId(): Future[ParticipantId] =
-    Future.successful(participantId)
+  override def getParticipantId(): Future[domain.ParticipantId] =
+    Future.successful(ledger.participantId)
```

Comment:
Isn't it a bit weird to get the participant identifier from something called `ledger`?

Reply:
Are you proposing to rename `Ledger`, `SqlLedger`, `ReadOnlyLedger`, ... to `*Participant`?